src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/_Layout.cshtml

```diff
@@ -88,5 +103,12 @@
         }
     }
     <persist-component-state />
+
+    <!-- Google Tag Manager (noscript) -->
+    <noscript>
+        <iframe src="https://www.googletagmanager.com/ns.html?id=@(googleTagContainerID)"
+                height="0" width="0" style="display:none;visibility:hidden"></iframe>
+    </noscript>
+    <!-- End Google Tag Manager (noscript) -->
```

Comment:
please remove this script tag. without javascript, the app itself is not going to work.